Source/Csla/Server/ActiveAuthorizer.cs

```diff
@@ -0,0 +1,75 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="ActiveAuthorizer.cs" company="Marimer LLC">
+//     Copyright (c) Marimer LLC. All rights reserved.
+//     Website: https://cslanet.com
+// </copyright>
+// <summary>Implementation of the authorizer that</summary>
+//-----------------------------------------------------------------------
+using Csla.Security;
+
+namespace Csla.Server
+{
+  /// <summary>
+  /// Implementation of the authorizer that
+  /// checks per-type authorization rules
+  /// for each request.
+  /// </summary>
+  public class ActiveAuthorizer : IAuthorizeDataPortal
+  {
+    /// <summary>
+    /// Creates an instance of the type.
+    /// </summary>
+    /// <param name="applicationContext"></param>
+    public ActiveAuthorizer(ApplicationContext applicationContext)
+    {
+      ApplicationContext = applicationContext;
+    }
+
+    private ApplicationContext ApplicationContext { get; set; }
+
+    /// <summary>
+    /// Checks authorization rules for the request.
+    /// </summary>
+    /// <param name="clientRequest">
+    /// Client request information.
+    /// </param>
+    public void Authorize(AuthorizeRequest clientRequest)
+    {
+      if (clientRequest.Operation == DataPortalOperations.Create)
```

Comment:
The code in the AuthorizeRequest class which does this job is more succinct. All we have to decide is if it less readable or less secure because of its brevity?

I was surprised to see the code in AuthorizeRequest; I would expect this to be a POCO, but it contains functionality. I'm wondering if there is any benefit to the shorter code in AuthorizeRequest; if there is then maybe we should use that here - and then maybe delete it from AuthorizeRequest